gremlin-go/docker/gremlin-server-integration.yaml

```diff
@@ -18,15 +18,16 @@
 host: 0.0.0.0
 port: 45940
 evaluationTimeout: 30000
+channelizer: org.apache.tinkerpop.gremlin.server.channel.UnifiedChannelizer
 graphs: {
-  graph: conf/tinkergraph-empty.properties,
-  immutable: conf/tinkergraph-empty.properties,
-  classic: conf/tinkergraph-empty.properties,
-  modern: conf/tinkergraph-empty.properties,
-  crew: conf/tinkergraph-empty.properties,
-  grateful: conf/tinkergraph-empty.properties,
-  sink: conf/tinkergraph-empty.properties,
-  tx: conf/neo4j-empty.properties
+  graph: scripts/tinkergraph-empty.properties,
```

Comment:
was this intentional? i thought `tinkergraph-empty.properties` was in in `conf/`: https://github.com/apache/tinkerpop/tree/master/gremlin-server/conf

Reply:
Yes, this was intentional. The `tinkergraph-empty.properties` in the `conf` folder is for regular server set up, the `tinkergraph-empty.propertie` for testing was in the `test/scripts` folder. 
It is more of an oversight before as the difference between them didn't have an impact in 3.5-dev, but for 3.6-dev, integration tests needed the `gremlin.tinkergraph.service` set up for server, so I switched to the `test/scripts` folder here.

Reply:
hmm - what is the difference? the files aren't identical?

Reply:
Right, the files are not identical. 
Example from 3.6, `.properties` in the `conf` only contains a simple empty graph https://github.com/apache/tinkerpop/blob/3.6-dev/gremlin-server/conf/tinkergraph-empty.properties, where as in the the `scripts` the Text Search and Degree Centrality services are enabled (which are needed for the cucumber `g.call()` tests) https://github.com/apache/tinkerpop/blob/3.6-dev/gremlin-server/src/test/scripts/tinkergraph-empty.properties.

Reply:
hmm - too many config/properties files. please make a note that unification would be ideal and that as you look to replicate this implementation pattern to the other language variants we should have that goal in mind. i actually think we might have missed something by not changing `gremlin-server/conf/tinkergraph-empty.properties` as that is the file that ships with the packaged Gremlin Server. would be nice if `call()` was ready to work there. none of this is a blocker for this PR.

Reply:
Sounds good, I will keep that in mind going forward. Thanks Stephen!